lib/BloqadeSchema/src/submit_to_braket.jl

```diff
@@ -57,14 +59,20 @@ between the original waveforms in `h` and the newly generated ones compatible wi
 """
 function submit_to_braket(h::BloqadeExpr.RydbergHamiltonian,
                           translation_params::BloqadeSchema.SchemaTranslationParams;
+                            validate_h = true,
                             arn="arn:aws:braket:us-east-1::device/qpu/quera/Aquila",
                             region="us-east-1",
                             credentials::Union{AWSCredentials, Nothing}=nothing
                         )
 
     # Transform error info is output via @debug
     h_transformed, _ = hardware_transform(h)
-    bloqade_ir = BloqadeSchema.to_schema(h_transformed, translation_params)
+    # bloqade_ir = BloqadeSchema.to_schema(h_transformed, translation_params)
+    bloqade_ir = if validate_h 
+        BloqadeSchema.to_schema(h_transformed, translation_params)
+    else
+        BloqadeSchema.to_schema_no_validation(schema_parse_rydberg_fields(h)..., translation_params)
```

Comment:
this `h` should be `h_transformed`

Reply:
Also, `schema_parse_rydberg_fields` returns an extra argument which `to_schema_no_validation` doesn't expect.